apps/console/src/extensions/components/authenticators/fido/quick-start.tsx

```diff
@@ -77,7 +77,7 @@ const FIDOQuickStart: FunctionComponent<FIDOQuickStartPropsInterface> = (
                         >
                             Choose the <Link external={ false } onClick={ () => setShowApplicationModal(true) }>
                                 application </Link>
-                            for which you want to set up FIDO2 login.
+                            for which you want to set up passkey login.
```

Comment:
Update the i18N files (french and sinhala) related to this key. Check other places as well.

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4156/commits/016d2dbfe86af4ef44d585dd2ef6e246e6ad0372